app/representers/api/v1/unclaimed_user_representer.rb

```diff
@@ -0,0 +1,13 @@
+module Api::V1
+  class UnclaimedUserRepresenter < Roar::Decorator
```

Comment:
That might be true. @jpslav should we assume that if a site creates an account for you and you login on that account, then the site is allowed to read your info? Or should we require explicit authorization (i.e. logging into that specific site)